core/src/main/java/de/danoeh/antennapod/core/util/playback/PlaybackController.java

```diff
@@ -521,13 +521,15 @@ public void onServiceQueried()  {}
      * Should be used by classes which implement the OnSeekBarChanged interface.
      */
     public float onSeekBarProgressChanged(SeekBar seekBar, int progress,
-                                          boolean fromUser, TextView txtvPosition) {
+                                          boolean fromUser, TextView txtvPosition, TextView seekDisplay) {
         if (fromUser && playbackService != null && media != null) {
             float prog = progress / ((float) seekBar.getMax());
             int duration = media.getDuration();
+            if (duration == -1) duration = getDuration();
```

Comment:
Can you just replace media.getDuration with getDuration? I think this should solve the issue too, without having to add checks.

Reply:
Sure I'll change it